tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -338,9 +338,9 @@ def genai_gen_visual_text(model, prompt, image, processor, tokenizer, max_new_to
     config.do_sample = False
     model.set_generation_config(config)
     if tokenizer.chat_template is not None:
-        model.start_chat(tokenizer.chat_template)
-    else:
-        model.start_chat()
+        model.get_tokenizer().set_chat_template(tokenizer.chat_template)
```

Comment:
why do we need to set chat template? `Tokenizer` should be aware of if and read from tokenizer_config.json

Reply:
@ilya-lavrenov, should I remove this line if it works underneath already?

Reply:
yes